lips/lip-10.md

```diff
@@ -0,0 +1,191 @@
+---
+lip: 10
+title: Libra ID Spec
+authors: Andrey Chursin(@andll), Kevin Hurley (@kphfb)
+status: Draft
+type: Informational
+created: 11/03/2020
+---
+# Summary
+
+This LIP describes Libra ID - the human-readable identifier for user accounts.   
+
+In addition to describing the format of the Libra ID, this LIP also describes the off-chain protocol for the peer-2-peer payment with Libra ID.
+
+This document does not describe an off-chain protocol for using Libra ID for merchant use case - this protocol will be part of merchant flow LIP.  
+
+# Motivation
+Libra ID provides a convenient way to identify user's account within a VASP. There are two main use cases for Libra ID:
+* For peer-2-peer payments, Libra ID allows a user to provide a human-readable ‘address’ to the sender. Basically, instead of sending a lip-5 address(that needs to be refreshed for each payment, ideally), the receiver can just share their Libra ID once. In this case Libra ID plays the role of an email address for payments. Benefits for this use case:
+  * Libra ID is a user-friendly identifier, unlike lip-5 address
+  * Libra ID resolution process can allow to exchange information about both sender and receiver for payment
+  * Libra ID resolution allows to fetch details about each participant, like profile picture
+  * Libra ID allows for better anonymity on-chain, as it is possible to share it once, but for wallet apps to transparently resolve it to new subaddress for each transaction. This is a big improvement comparing to exchange of lip-5 identifiers, that leaks privacy information on the chain and allows to track all transactions for the individual subaccount.
+* For the merchant use case, Libra ID allows users to self-identify themself to merchants. In this case Libra ID plays the role of credit card number or also can be seen as a variant of OpenID-type protocol.
+
+## Examples in other blockchains
+Ripple proposes PayID. PayID has some similarity to the current proposal, but is not entirely the same thing.
+For example, PayID seems to target cross-chain payments, while libra id is supposed to facilitate cross-vasp payments, but only inside the libra network.
```

Comment:
I wonder if we want to mention that the initial target for LibraID is within Libra, but that nothing in the design specifically prevents cross-chain.  And in fact, we should actually think through to make sure that is the case.  I could see it potentially being useful to go across chains at some point.  Maybe the lookup still needs to happen on Libra, but in the off-chain messages, maybe we can later add a chain identifier that coudl be eth or bitcoin, etc.  I don't think this is something we should build now, but I wonder if we can feel pretty certain that this would be possible in the future if we later desired to do so